source/adapters/cuda/ur_interface_loader.cpp

```diff
@@ -303,6 +303,20 @@ UR_DLLEXPORT ur_result_t UR_APICALL urGetCommandBufferExpProcAddrTable(
   return retVal;
 }
 
+UR_DLLEXPORT ur_result_t UR_APICALL urGetEventExpProcAddrTable(
+    ur_api_version_t version, ///< [in] API version requested
+    ur_event_exp_dditable_t
+        *pDdiTable ///< [in,out] pointer to table of DDI function pointers
+) {
+  auto retVal = validateProcInputs(version, pDdiTable);
+  if (UR_RESULT_SUCCESS != retVal) {
```

Comment:
Nit: Prefer regular `lhs_val == const_rhs_val`, i.e. retVal != UR_RESULT_SUCCESS over  the Yoda notation, for readability reasons. This is not a blocker.

Reply:
I'm a little confused by your comments (this one and following), because all the conditions in these files use Yoda notation.